null_safety_examples/null_safety_codelab/bin/late_keyword.dart

```diff
@@ -0,0 +1,13 @@
+class Meal {
+  late String description;
+
+  void setDescription(String str) {
```

Comment:
It seems very not "Dart" like to include a setter method like this. Perhaps you could switch this example to use a getter and setter:

Late variables aren't generally a good idea to expose publicly anyway as you don't have much control over what others will do.

```dart
class Meal {
  late String _description;

  void set description(String str) => _description = str;

  String get description => _description;
}

void main() {
  final myMeal = Meal();
  myMeal.description = 'Feijoada!';
  print(myMeal.description);
}
```

Reply:
Goodness, this sounds reasonable but we might need to think about it. I'll add it to that follow-up issue.